From: Andreas Tille <tille@debian.org>
Date: Sat, 21 May 2016 09:41:25 +0200
Subject: format security

Test breaks format security and should be fulfilled on Linux anyway.
(See discussion on debian-mentors list in May 2016)
---
 arithchk.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arithchk.c b/arithchk.c
index a5fb746..074e468 100644
--- a/arithchk.c
+++ b/arithchk.c
@@ -138,6 +138,7 @@ ccheck(int ac, char **av)
 
 	/* Cray1 = 4617762693716115456 -- without overflow on non-Crays */
 	/* The next three tests should always be true. */
+/* Skip useless test
 	Cray1 = ac >= -2 ? 4617762 : 0;
 	if (ac >= -1)
 		Cray1 = 1000000*Cray1 + 693716;
@@ -146,6 +147,7 @@ ccheck(int ac, char **av)
 	u.d = 1e13;
 	if (u.L == Cray1)
 		return &CRAY;
+*/
 	return 0;
 	}
 
